### PR TITLE
feature/SOF-3636

### PR DESCRIPTION
**Pull request by mohammadidinani**

### CODE REVIEW

File: src/cell/conventional_cell.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    ],
+
+};
+
+/**
+ * Returns lattice vectors for a conventional cell for a lattice.
+ * @param {Lattice} lattice - Lattice instance.
+ * @return {Array[]} Cell.vectorsAsArray
+ */
+export function conventionalCell(lattice) {
```
Comment (timurbazhirov) [inline src/cell/conventional_cell.js]: getConventionalCellFromPrimitiveLattice

File: src/cell/conventional_cell.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    [LATTICE_TYPE.HEX]: unitMatrix,
+
+    [LATTICE_TYPE.RHL]: unitMatrix,
+
+    [LATTICE_TYPE.MCL]: unitMatrix,
+
+    [LATTICE_TYPE.MCLC]: [
+        [1, -1, 0],
+        [1, 1, 0],
+        [0, 0, 1]
```
Comment (timurbazhirov) [inline src/cell/conventional_cell.js]: Are you sure about this one and ORCC above? Just doublechecking

File: src/cell/conventional_cell.js (hunk 1 above)
Comment (mohammadidinani) [reply src/cell/conventional_cell.js]: Renamed.

File: src/cell/conventional_cell.js (hunk 2 above)
Comment (mohammadidinani) [reply src/cell/conventional_cell.js]: It seems as they have the same representations.

**Pull request merged**